src/app/globals.css

```diff
@@ -214,6 +218,14 @@
 .mapboxgl-ctrl-legend-pane ul.list li:last-child {
   display: none;
 }
+
+/* shift map search bar down and to the left 20px */
+
+#main .maplibregl-ctrl-top-right {
+  top: 20px;
+  right: 20px;
+}
+
```

Comment:
I don't believe these css changes are in scope with your PR, please feel free to remove